src/values/AbstractValue.js

```diff
@@ -133,9 +128,10 @@ export default class AbstractValue extends Value {
   addSourceLocationsTo(locations: Array<BabelNodeSourceLocation>, seenValues?: Set<AbstractValue> = new Set()): void {
     if (seenValues.has(this)) return;
     seenValues.add(this);
-    if (this._buildNode && !(this._buildNode instanceof Function)) {
-      if (this._buildNode.loc) locations.push(this._buildNode.loc);
-    }
+    // TODO: make this work again?
+    // if (this._buildNode && !(this._buildNode instanceof Function)) {
+    //   if (this._buildNode.loc) locations.push(this._buildNode.loc);
+    // }
```

Comment:
What was this about?

Reply:
Removing it in follow up, it was a really early comment from the first day on that PR. Thanks for pointing it out.